sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,194 @@
+---
+id: high-contrast
+title: High Contrast in FAST
+sidebar_label: High Contrast
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/high-contrast.md
+---
+
+## Styling components using forced-colors.
+High contrast mode uses the CSS media feature, `forced-colors`. When `forced-colors` is set to active, the user agent will apply a limited color palette to the component.
+
+**Example:**
+```css
+@media (forced-colors: active) {
+    :host {
+        background: ButtonFace;
+    }
+}
+```
+
+FAST has a utility function that is use to construct `forced-colors` in the stylesheet, called [forcedColorsStylesheetBehavior()](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/utilities/match-media-stylesheet-behavior.ts). This is added in the `withBehavior()` function inside the `css` style.
+
+**Example**
+```css
+export const ComponentStyles = css`
+    /* ... */
+ `.withBehaviors(
+    forcedColorsStylesheetBehavior(
+        css`
+            :host {
+                background: ButtonFace;
+            }
+        `
+    )
+);
+```
+
+## System Color Keyword
+In `forced-colors` mode, the colors on the component are reduced to a limited color pallete chosen by the user. The system color keyword exposes these user chosen colors.
+
+Below are the system color keywords we use in FAST. The keywords are used as color values on style properties.
+
+
```

Comment:
I think we should probably just link to https://developer.mozilla.org/en-US/docs/web/css/color_value#System_Colors instead of hosting this information ourselves.